docxcompose/composer.py

```diff
@@ -244,8 +245,8 @@ def add_styles_from_other_parts(self, doc):
     def add_styles(self, doc, element):
         """Add styles from the given document used in the given element."""
         our_style_ids = [s.style_id for s in self.doc.styles]
-        used_style_ids = set([e.val for e in xpath(
-            element, './/w:tblStyle|.//w:pStyle|.//w:rStyle')])
+        used_style_ids = list(OrderedDict.fromkeys([e.val for e in xpath(
```

Comment:
this is the same as
```python
[e.val for e in xpath(element, './/w:tblStyle|.//w:pStyle|.//w:rStyle')]
```
you can skip creating an `OrderedDict` and building a list of its keys by just using the initial list 😄.

Reply:
as discussed that makes sense, of course 👍